### PR TITLE
[Hot fix] Prevent multiple concurrent analytics refreshes

### PR DESCRIPTION
**Pull request by edmofro**
I've listed this as a hot fix as it'd be great to get it into production to help make Juliana's job importing heaps of EMIS data easier, though we do still need to test it on its own deployment and see whether it will actually help with that

### CODE REVIEW

**Review by rohan-bes — APPROVED** [review]
Looks great! And love the unit tests!

File: packages/database/src/analytics/AnalyticsRefresher.js [hunk 1]
```diff
@@ -33,7 +35,10 @@ export class AnalyticsRefresher {
     return this.scheduleAnalyticsRefresh();
   };
 
-  scheduleAnalyticsRefresh() {
+  async scheduleAnalyticsRefresh() {
+    // wait for any active refresh to finish before scheduling a new one
+    await this.activeRefreshPromise;
```
[the remark this answers is not in the document]
Comment (edmofro) [reply packages/database/src/analytics/AnalyticsRefresher.js:40]: Ha yeah luckily not! Comes down to a nice existing `AnalyticsRefresher` class :-)

Yep, I think the manual refreshes can be handled if and when they become a problem. The admin panel idea would be a good solution.

File: packages/database/src/tests/analytics/AnalyticsRefresher.test.js [hunk 2]
```diff
@@ -30,10 +32,12 @@ const matchingFields = [
   'date',
 ];
 
+const REFRESH_DEBOUNCE_TIME = 100; // short debounce time so tests run more quickly
```
[the remark this answers is not in the document]
Comment (edmofro) [reply packages/database/src/tests/analytics/AnalyticsRefresher.test.js:35]: Yeah good suggestion but I'll leave that for a rainy day 😛

**Comment by edmofro** [comment]
Tested with @julianam-w and it was refreshing analytics tables correctly, and only once at a time. There were some other issues with the Samoa data, but that seems unrelated. Merging this in as a hot fix.

**Pull request merged**